pipelines/tekton/aiedge-e2e/aiedge-e2e.pipeline.yaml

```diff
@@ -30,6 +30,10 @@ spec:
   - name: modelRelativePath
     type: string
     description: "Location of the model within the context of the source location.  Leave blank if the model files are at the root of the source location."
+  - name: model-dir
+    type: string
+    description: Directory below <modelRelativePath>/<model-name>/ to be used as MODEL_DIR in image build.
```

Comment:
We need to revisit this in a separate PR/Jira as effectively we now have 2 relative paths :) But I think an issue for that already exists, I will check.

Reply:
Yes, as mentioned in the commit message for this one, it's kind of a hack because of something that was hardcoded in the Containerfile. IMHO it's a symptom of having one uber-generic pipeline to try to cover all types of models and model servers.